usr/src/cmd/acct/acctprc.c

```diff
@@ -62,6 +62,7 @@ void **root = NULL;
 
 void output(void);
 void enter(struct ptmp *);
+char *uidtonam();
```

Comment:
```suggestion
extern char *uidtonam(uid_t);
```
(I know the other files that you aren't touching use the K&R form, but let's fix this line here).

and now that the prototype is there, you can drop the explicit cast to `(char *)` where it is used, in `print_node()`.